tests/test_oidc.py

```diff
@@ -10,6 +10,8 @@
 
 from ansys.openapi.common import ApiClientFactory
 from ansys.openapi.common._oidc import OIDCSessionFactory
+from ansys.openapi.common import _oidc
```

Comment:
Can we either import exactly what we need, or use the _oidc namespace throughout?